lib/mobility/backend.rb

```diff
@@ -1,4 +1,6 @@
 # frozen-string-literal: true
+require 'digest/md5'
```

Comment:
I'm a bit unsure whether this is ok. Any application that uses Mobility will now require md5. I just want a short unique hash for the options hash... :thinking:

Reply:
I would say it is not a big deal since this is part of Ruby Standard Library.